### PR TITLE
Changed normalization parameters for CIFAR-10 images

### PR DESCRIPTION
**Pull request by ljaljushkin**
### Changes

Changed mean/std values for pre-processing input images from CIFAR10 dataset

### Reason for changes

It helps to reuse models pretrained on CIFAR10 from:
https://github.com/huyvnphan/PyTorch_CIFAR10/blob/master/data.py#L16
This models are used for e2e validation of BootstrapNAS feature #1079 

### Tests

It affects backward compatibility tests that compare actual test accuracy with 'best_acc1' value in the checkpoint.
The `best_acc1` value was overridden to correspond to the introduced pre-processing.

### CODE REVIEW

**Comment by ljaljushkin** [comment]
started custom build 29 to run backward_compatibility tests with patched checkpoints.

**Comment by vshampor** [comment]
The single failure will be fixed by updating the internal checkpoints.

**Review by vshampor — APPROVED** [review]
